Program.cs

```diff
@@ -267,16 +264,17 @@ static string OneCityValidator(Statistics cityStats)
             string response;
             do
             {
-                Console.Write("Enter city name or type 'list' to display all cities: ");
+                Console.Write("\nEnter city name or type 'list' to display all cities: ");
                 response = Console.ReadLine();
+                response = response.Trim();
                 if (response.ToLower() == "list")
                 {
                     Console.Write("\n Enter the province name to see the cities located in that given province: ");
```

Comment:
I think we need to ALSO add an option to list the provinces for full hand hold mode. That can be done in a separate PR though

Reply:
We could just display the list of provinces then say "pick a province to display"

Reply:
Yeah I agree